wimhofbreather.py

```diff
@@ -0,0 +1,80 @@
+import vlc
+import time
+import datetime
+import os
+import sys
+import random
+
+
+class WimHofBreather:
+    testing_mode = True
+    breathing_times = []
+    random.seed(datetime.datetime.now())
+
+    def __init__(self):
+        print("Alright, guys!")
+        self.breathing_sets = int(input("How many breathing sets would you like to do?\n"))
+        self.script_dir = os.path.dirname(__file__)
+        if len(sys.argv) > 1 and sys.argv[1] == "debug":
+            self.testing_mode = True
+
+    def get_breathing_times(self):
+        return self.breathing_times
+
+    def lets_do_some_breathing(self):
+
+        for index in range(1, self.breathing_sets + 1):
+
+            self.play_breathing_audio(index)
+            self.play_music_audio(index)
+            self.play_holding_audio()
+
+    def play_breathing_audio(self, index):
+        if index > 3:
+            index = 'n'
+        if self.testing_mode:
+            wim_player = vlc.MediaPlayer(os.path.join(self.script_dir, 'audio/wim/breathing_testing.m4a'))
+        else:
+            wim_player = vlc.MediaPlayer(os.path.join(self.script_dir, f'audio/wim/breathing-{index}.mp3'))
+
+        wim_player.play()
+        time.sleep(1)  # Wait for audio to load and start playing
+
+        while wim_player.is_playing():
+            pass
+
+    def play_music_audio(self, index):
+
+        # Get a random song
+        song_list = [f for f in os.listdir(os.path.join(self.script_dir, f"audio/song{index}")) if
+                     f.endswith('mp3')]
+        song = os.path.join(self.script_dir, f'audio/song{index}/' + random.choice(song_list))
+        music_player = vlc.MediaPlayer(song)
+
+        music_player.audio_set_volume(70)  # Wim doesn't talk very loud
+        music_player.play()
+
+        time_breathing = self.record_time_with_sentinel()
+        self.breathing_times.append(time_breathing)
+
+        if music_player.is_playing():
+            music_player.stop()
+
+    def play_holding_audio(self):
```

Comment:
rename `play_holding_audio` to `play_15_second_breath_hold`

Reply:
done, and comment removed